oat-sa/extension-tao-itemqti
# Feature/AUT-17/blockAdder plugin: updated styles for plus button

Related to: https://oat-sa.atlassian.net/browse/AUT-17
Related to: https://github.com/oat-sa/extension-tao-mediamanager/pull/255

How to test:
- go to Passage Authoring
- check plus button

![image](https://user-images.githubusercontent.com/25976342/108960018-52190a00-7686-11eb-84cb-6faba538efd0.png)
